CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# Samply.Beam 0.8.0 - 2024-07-26
+
+This major release mostly features some bugfixes and the big upgrade of `hyper` to version 1.
+
+## Breaking changes
+
+* The `filter=todo` parameter on the tasks endpoint will now not yield tasks anymore which are claimed. This was bug but does change the behavior of the tasks endpoint in some cases.
+* beam lib reexports types from `reqwest` which has been upgraded to the hyper 1 compatible version. This may lead to compile errors in code which uses old reqwest versions.
```

Comment:
```suggestion
* beam lib reexports types from `reqwest` which has been upgraded to the hyper 1 compatible version. This may lead to compile errors in code bases using older reqwest versions.
```